src/main/java/com/manardenza/dao/HotelDaoImpl.java

```diff
@@ -1,14 +1,20 @@
 package com.manardenza.dao;
 
 import com.manardenza.entity.Hotel;
+import com.manardenza.utils.DatabaseSeeder;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.manardenza.utils.DatabaseSeeder'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)